### ISSUE TITLE
Compile problem - aggregate root, primary key with more than 5 fields

### ISSUE THREAD

**Issue by Kobus-Smit** [ngs-doo/revenj#96]
A dsl with >5 fields primary keys does not compile.

Clicking the compile button shows the `Diffing Postgres DSL...` for a few ms and then nothing happens.

```c#
module Test {

aggregate XYZ(A, B, C, D, E, F) {
  int A;
  int B;
  int C;
  int D;
  int E;
  int F;
}

}
```

Using the DDD plugin in Visual Studio 2017 v15.5.7
Compiler v1.9.6592.23874
PostgreSQL v10

**Comment by zapov** [comment]
Tnx for the bug report. This should be fixed now (at least the nothing happens part)

**Comment by Kobus-Smit** [comment]
Resolved with Compiler v2.0.6627.22972
Great thanks @zapov !
